OptionSelector/CategoryTableViewController.m

```diff
@@ -0,0 +1,118 @@
+//
+//  CategoryTableViewController.m
+//  OptionSelector
+//
+//  Created by Henna on 8/10/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CategoryTableViewController.h"
+#import "CQCategory.h"
+#import "OptionsTableViewController.h"
+#import "CategoryModel.h"
+
+
+@interface CategoryTableViewController ()
+@property (nonatomic) CategoryModel * data;
```

Comment:
You don't really need to store a reference to the model because it's a singleton. It would be better to make this a dynamic readonly property that returns the shared instance. For example:
```objc
@interface CategoryTableViewController ()

@property (nonatomic, readonly) CategoryModel *data;

@end

@implementation CategoryTableViewController

- (CategoryModel *)data
{
    return [CategoryModel sharedInstance];
}
```